GroceryStore/ProductTableView.h

```diff
@@ -0,0 +1,13 @@
+//
+//  ProductTableView.h
+//  GroceryStore
+//
+//  Created by Jilian Scott on 5/13/15.
+//  Copyright (c) 2015 Vokal. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+@interface ProductTableView : UITableView
```

Comment:
I think I mentioned this somewhere above, but if you're not doing any customization in the code, you don't need to subclass UITableView.